.github/ISSUE_TEMPLATE/3.1.1.md

```diff
@@ -0,0 +1,33 @@
+---
+name: 3.1.1. Pre-Lecture Quiz
+about: Terrarium / Introduction to HTML
+title: '3.1.1. Pre-Lecture Quiz'
+labels: '3. Terrarium, Pre-lecture quiz'
+
+---
+# Terrarium Project Part 1: Introduction to HTML
+
+![Introduction to HTML](https://github.com/Extenza-Academy/WebDev-100_2021-Q1/blob/main/lessons/3-terrarium/1-intro-to-html/images/webdev101-html.png)
+> Sketchnote by [Tomomi Imura](https://twitter.com/girlie_mac)
+
+## Pre-Lecture Quiz
+
+*A warm-up quiz about HTML*
+
+Complete this quiz in class
+
+1. HTML stands for 'HyperText Mockup Language'
+
+- [ ] [true]
```

Comment:
Square brackets around answers and questions are unnecessary in my opinion.
@mserykh, could you please remove them in all pre- and post-lecture quizes?

> I know that you didn't put them there (they were in original lessons).

Reply:
@dev-experience done